integrationtests/tests/fixtures/mod.rs

```diff
@@ -559,7 +559,7 @@ async fn distributed_config(
             let cfg = ServerConfig::distributed_gen(
                 &our_params,
                 registry.legacy_init_modules(),
-                DelayCalculator::TEST_DEFAULT,
```

Comment:
OK, this is so subtle yet notable difference, that I don't think we should leave it as is.

I mean - `TEST_DEFAULT` was supposed to be ... the default... for the tests... and this is a test file... :D

IMO:

* Maybe we should get rid of the `Default::default()` and `DelayCalculator::new()` on this type; too easy to use them carelessly. I don't think `Default` is needed, and we can have `DelayCalculator::production()` or something?
* Maybe make this exponential or give it a threshold when to stop being so aggressive? It's great that it tries fast initially, but the floods of reconnection logs when something doesn't work well is a problem. @douglaz , any thoughts?

Reply:
Exponential backoff would be great! But this PR already seems to be an improvement.

Reply:
The problem with the "production" delay is due to its exponential nature it may lead to very high "tail latencies". By very high I don't mean 5% more in general, but like 2x-3x more 1-5% of time. At least this was true when I tested. This may have changed. Perhaps a better way would to make the "test" delay a fixed (like 1s)

Reply:
> Good find!
> 
> OK, this is so subtle yet notable difference, that I don't think we should leave it as is.
> 
> I mean - `TEST_DEFAULT` was supposed to be ... the default... for the tests... and this is a test file... :D
> 
> IMO:
> 
> * Maybe we should get rid of the `Default::default()` and `DelayCalculator::new()` on this type; too easy to use them carelessly. I don't think `Default` is needed, and we can have `DelayCalculator::production()` or something?

But yes, I this is behavior is okay I would just increase the scaling factor for this non production parameter profile instead of switching to the production one. Perhaps the `default` implementation is really misleading here.

Reply:
> The problem with the "production" delay is that due to its exponential nature it may lead to very high "tail latencies".

I don't understand this one.

Usually what I'd implement is exponential + ceiling (like 10/30/60s). So it does 1ms,2ms,4,8 and eventually just constant 10s.

Reply:
https://github.com/fedimint/fedimint/issues/2263

Reply:
Yep always have a ceiling for exponential backoffs.

Reply:
>  The problem with the "production" delay is that due to its exponential nature it may lead to very high "tail latencies". By very high I don't mean 5% more in general, but like 2x-3x more 1-5% of time. At least this was true when I tested. This may have changed. Perhaps a better way would to make the "test" delay a fixed one (like 1s)

That sounds kinda ok, the tests would take a bit longer but still succeed?

Reply:
Tests were about 20s slower with this, but we should cap the timeout just in case, don't think it's capped now.